app/assets/typescript/index.ts

```diff
@@ -10,6 +10,7 @@ import { CubeCoordinates } from "hexagrid";
 import { buildingName } from "../../data/building";
 import { factionColor } from "../../graphics/utils";
 import { Booster } from "@gaia-project/engine/src/enums";
+import boosts from "@gaia-project/engine/src/tiles/boosters";
```

Comment:
please make sure to export things like that in `index.ts` of gaia-engine

Reply:
ok

Reply:
I need your advice on this. I had to rename boosters from tiles/boosters because they where conflicting with the different boosters in code. WHat's the best practice in this case. I would change the originala boosters to boosterTiles and then use boosters when we are referring to a subset of those tiles

Reply:
`This is `index.ts` of `gaia-engine`:

```ts
import Engine from "./src/engine";
import Player from "./src/player";
import PlayerData from "./src/player-data";
import { FactionBoard } from "./src/faction-boards";
import factions from "./src/factions";
import Event from "./src/events";
import AvailableCommand from "./src/available-command";

export {Condition, Planet, Resource, Operator, ResearchField, Faction, Command, Building, Booster} from "./src/enums";
export {GaiaHexData} from "./src/sector";

export { Player, PlayerData, Event, FactionBoard, factions, AvailableCommand };

export default Engine;
```

There's no 'boosters', so you can import & export it in gaia-engine. Then you can do `import {boosters} from '@gaia-project/engine`.